src/main/java/org/jabref/gui/actions/StandardActions.java

```diff
@@ -195,6 +195,8 @@ public enum StandardActions implements Action {
     GROUP_REMOVE_WITH_SUBGROUPS(Localization.lang("Also remove subgroups")),
     GROUP_CHAT(Localization.lang("Chat with group")),
     GROUP_EDIT(Localization.lang("Edit group")),
+    GROUP_GENERATE_EMBEDDINGS(Localization.lang("Generate embeddings for linked files in the group")),
+    GROUP_GENERATE_SUMMARIES(Localization.lang("Generate summaries for entries in the group")),
```

Comment:
It is interesting that you ordered it that way here - but the tabs in the entry editor are the other way round.

For consistency, I would swap these two commands. Always summaries first. (Or swap everywhere)